javascript/ql/test/query-tests/Security/CWE-079/DomBasedXssWithResponseThreat/wrapper.js

```diff
@@ -0,0 +1,2 @@
+import { useQuery } from "@tanstack/react-query";
+export { useQuery}
```

Comment:
```suggestion
export { useQuery }
```
Sorry, can't unsee the missing space here 😅